### PR TITLE
User can logout 

### PR DESCRIPTION
**Pull request by yasmineezequiel**
# User can logout
[PT-link](https://www.pivotaltracker.com/story/show/169765088)

## PR User story
```
As a user,
In order to keep my account safe 
I would like to be able to logout. 
```

## PR Content
- Logout functionality using redux token auth
- Acceptance tests

### CODE REVIEW

**Review by miyeaier — APPROVED** [review]

**Review by clalil — APPROVED** [review]

File: src/components/Logout.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        </Menu.Item>
+      </>
+    )
+  }
+}
+
+const mapStateToProps = state => {
+  return {
+    state: state,
+    currentUser: state.reduxTokenAuth.currentUser
```
Comment (oliverochman) [inline src/components/Logout.jsx:30]: do we need this state in this component?

File: src/components/Navbar.jsx [hunk 2]
```diff
@@ -11,6 +13,23 @@ class Navbar extends Component {
 
   render() {
     const notMobile = { minWidth: Responsive.onlyMobile.maxWidth + 1 }
+    let logOut, logIn
+
+    if (this.props.currentUser.isSignedIn) {
```
Comment (oliverochman) [inline src/components/Navbar.jsx:18]: I would turn this into an `if else` or a ternary operator

**Review by oliverochman — CHANGES_REQUESTED** [review]

File: src/components/Navbar.jsx (hunk 2 above)
Comment (clalil) [reply src/components/Navbar.jsx:18]: We agree! :)

File: src/components/Logout.jsx (hunk 1 above)
Comment (clalil) [reply src/components/Logout.jsx:30]: Nope, that does indeed seem redundant.

**Review by oliverochman — APPROVED** [review]

**Pull request merged**